util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java

```diff
@@ -135,8 +144,7 @@ public void updateTrustCredentials(X509Certificate[] trustCerts) throws IOExcept
       keyStore.setCertificateEntry(alias, cert);
       i++;
     }
-    X509ExtendedTrustManager newDelegateManager = createDelegateTrustManager(keyStore);
-    this.delegateManager = newDelegateManager;
+    this.delegateManager = createDelegateTrustManager(keyStore);
```

Comment:
Pre-existing readability nit: `this.` is redundant here and can be removed. There seem to be a few other instances of this pattern in this file, feel free to clean them all up if you like.

Reply:
Agreed, it's redundant here and the G style guide suggest to omit it.
I code it that way because it was the style here and at KeyManager, plus it also exists at
https://github.com/grpc/grpc-java/blob/8aaace12eb8697a2dd841d2261b0051a1414f0c8/api/src/main/java/io/grpc/TlsServerCredentials.java#L326

Let's defer to Eric

Reply:
That is the style of the original author, so don't change it willy-nilly. "Stay consistent." I prefer the `this` in larger classes, and really consider it a language mistake to support the implicit this, although I'll still vary my usage. I don't see anything in any style guide that suggests to omit it.

Reply:
Thank you, left the current version with `this.`